Sample for HIP training course

LakshmiKumar23 opened the pull request:
Using only brightness as an example from the test suite for hip training course.

asalmanp reviewed (commented):
@LakshmiKumar23 can you please move the entire examples folder under utilities folder?

LakshmiKumar23 commented:
> @LakshmiKumar23 can you please move the entire examples folder under utilities folder?

@asalmanp Also added a REAME and fixed codacy issues in the same commit
